EventFilter/SiStripRawToDigi/plugins/SiStripDigiToRaw.cc

```diff
@@ -335,7 +340,7 @@ namespace sistrip {
           //for special mode premix raw, data is zero-suppressed but not converted to 8 bit
           //zeroSuppressed here means converted to 8 bit...
           if (mode_ == READOUT_MODE_PREMIX_RAW) zeroSuppressed=false;
```

Comment:
This line is not needed any more (as well as the comment above, or it could be more conveniently moved near to the definition of ```const bool dataIsAlready8BitTruncated```, if you really want to maintain it)